acms-react-app/src/Services/auth.service.js

```diff
@@ -1,50 +1,30 @@
 import axios from 'axios';
 
-const API_URL = 'http://localhost:8080/api/v1/auth/';
+const API_URL = `${process.env.REACT_APP_BACKEND_URL || 'http://localhost:8080'}/api/v1/auth/`;
```

Comment:
I believe you forgot to change the http://localhost:8080 to the environment variable that you created. Make sure to use the environment variable